lib/model.js

```diff
@@ -214,13 +214,6 @@ _.assign(props, {
   mongodb_database_name: {
     type: 'string',
     default: undefined
-  },
-  /**
-   * Whether BSON values should be promoted to their JS type counterparts.
-   */
-  promote_values: {
-    type: 'boolean',
-    default: true
```

Comment:
this removes the ability to ever set promote_values via the connection model. 

What if I have an application that always wants to set it to `false`? It is convenient to be able to set it once for the entire connection and not at the query level anymore. 

Could you try if it suffices to not set the default to `true`, but keep the option there? I.e. set the default to `undefined` here, and in line 689, have a check 

```
if (this.promote_values !== undefined) {
  opts.db.promoteValues = this.promote_values;
)
```

Can you test if that would also work?